src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -193,6 +238,9 @@
         this.flushSizeOverrideMB = flushSizeOverrideMB;
         this.currentFlushSize = flushSizeOverrideMB << 20;
         this.expiredSSTableCheckFrequency = TimeUnit.MILLISECONDS.convert(expiredSSTableCheckFrequency, TimeUnit.SECONDS);
+        this.baseShardCount = baseShardCount;
+        this.targetSSTableSizeMin = targetSStableSize * Math.sqrt(0.5);
```

Comment:
No. This is just a representation of the target size which is somewhat more useful in the calculations. When we use the min and take the exponent (or highest one bit), it has the effect of rounding the logarithm up instead of truncating. See the comment in `getNumShards`.

I'm still not certain if we shouldn't just store the target and apply the multiplier in context in `getNumShards`.